SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialPackageMapClient.cpp

```diff
@@ -157,7 +157,7 @@ bool USpatialPackageMapClient::ResolveEntityActor(AActor* Actor, Worker_EntityId
 
 	if (GetEntityIdFromObject(Actor) == SpatialConstants::INVALID_ENTITY_ID)
 	{
-		UE_LOG(LogSpatialPackageMap, Error, TEXT("ResolveEntityActor failed for Actor: %s with NetGUID: %s"), *Actor->GetName(), *NetGUID.ToString());
+		UE_LOG(LogSpatialPackageMap, Error, TEXT("ResolveEntityActor failed for Actor: %s with NetGUID: %s and passed entity ID; %lld"), *Actor->GetName(), *NetGUID.ToString(), EntityId);
```

Comment:
Now seem to be triggering this for client owned entities a fair amount. Not sure why this change would do that, any input appreciated

Reply:
Fixed- there was an implicit dependency on the ordering of ops throughout the remove actor flow which required the entity to have been removed, but not had its components cleaned up yet. Now this uses the metadata component instead, I wasn't enforcing the metadata component to still be around. Moving the remove actor call above the remove metadata component call fixes the issue. We may want to refactor the package client to make dependencies like these more explicit.